### PR TITLE
Added support for CFScript

### PR DESCRIPTION
**Pull request by mjclemente**
This partially deals with #2035, though does not resolve it entirely.

CFML has independent tag and script based syntaxes. This adds support for script syntax.

Here are some language references:

- https://cfdocs.org/script
- https://helpx.adobe.com/coldfusion/developing-applications/the-cfml-programming-language/extending-coldfusion-pages-with-cfml-scripting/the-cfscript-language.html
- https://www.petefreitag.com/cheatsheets/coldfusion/cfscript/

### CODE REVIEW

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Good work @mjclemente!

A couple of minor things and then this is ready.

**Comment by mjclemente** [comment]
Thanks for the fast, thorough, and really helpful review 😀 

I believe I've made all requested changes.

File: components.json [hunk 1]
```diff
@@ -238,6 +238,12 @@
 			"require": "c",
 			"owner": "zeitgeist87"
 		},
+		"cfscript": {
+			"title": "CFML Script",
```
Comment (RunDevelopment) [inline components.json:242]: The docs consistently call this langauge "CFScript". I think we should too.
```suggestion
			"title": "CFScript",
```

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Thank you for the quick response!

One more comment. Also, could you please add the link to https://cfdocs.org/script in a comment at the beginning of the file? This will be helpful to future contributors.

**Comment by mjclemente** [comment]
Makes sense!

Changes made. Again, thanks for the prompt and thorough handling!

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
Thank you for contributing @mjclemente!

**Comment by mjclemente** [comment]
Thank you! 🎉

**Comment by dawesi** [comment]
can we also create a branch of cfml for cfml/lucee (which boxlang also supports that syntax)

This is divergent from Adobe CFML/CFScript now also, as it containers many functions / helpers methods that are not in coldfusion and vice versa.

They publish the latest docs in json files zipped here.

https://docs.lucee.org/lucee-docs-json.zip

**Comment by mjclemente** [comment]
Given that Lucee/Boxlang/Adobe ColdFusion are all CFML, my first thought would be that they could all be incorporated here. While there may be some different tags, I don't see any issues in including them all - they have more in common than different (at least Lucee CFML and Adobe). Not sure what functions are missing at this point.

**Pull request merged**